src/scripts/logBook/LogBookTypes.ts

```diff
@@ -24,4 +24,12 @@ const LogBookTypes: Record<string, LogBookType> = {
         display: 'primary',
         label: 'FOUND',
     },
+    ACHIEVEMENT: {
+        display: 'success',
+        label: 'ACHIEVE',
+    },
+    QUEST_COMPLETE: {
+        display: 'success',
+        label: 'QUEST',
+    },
```

Comment:
Maybe we could use something like `warning` and `info` here instead to make them stand out from the rest of the logs:
```suggestion
    ACHIEVEMENT: {
        display: 'warning',
        label: 'ACHIEVE',
    },
    QUEST_COMPLETE: {
        display: 'info',
        label: 'QUEST',
    },
```

Before | After
:-- | :--
![image](https://user-images.githubusercontent.com/7288322/97380970-fa170480-192c-11eb-975c-f722c1ffe2b1.png) | ![image](https://user-images.githubusercontent.com/7288322/97381013-12871f00-192d-11eb-8865-ec016d62fa09.png)

Reply:
Looks sleeker with the `warning` and `info` colours